src/components/ComposerEditor/ComposeEditor.spec.tsx

```diff
@@ -1,16 +1,31 @@
-import { render } from '@testing-library/react';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
 
 import ComposerEditor from './ComposerEditor';
 
-const placeholderText = 'Digite algo aqui...';
-
 describe('ComposeEditor', () => {
   describe('when render component', () => {
-    it('successfully renders with a TextArea', () => {
-      const { getByPlaceholderText } = render(<ComposerEditor />);
-      const textArea = getByPlaceholderText(placeholderText);
+    it('should mount a textbox', () => {
```

Comment:
Should é redundante e repetitivo, pode remover.

```suggestion
    it('mount a textbox', () => {
```